### ISSUE TITLE
Problems view do not show markers when Show Active File filter is on

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#99155]
Testing #98821

- Enable the filter `Show Active File Only` in Problems view using the filter dropdown in filter input box
- Open any notebook with problems

🐛 Problems are not shown

<img width="1346" alt="image" src="https://user-images.githubusercontent.com/10746682/83549389-8e5b7780-a505-11ea-9682-894c69534932.png">

**Comment by sandy081** [comment]
@jrieken Sorry this is my bad that I forgot about this feature when we discussed.

**Comment by jrieken** [comment]
Yeah, this isn't so easy because cell have uris like this `vscode-notebook-cell:///some/path/notebook.file#42` while the notebook itself is `file:///some/path/notebook.file`... Because of the different schemes it's hard to tell what the right file is
